collect_app/src/main/java/org/odk/collect/android/tasks/sms/models/SentMessageResult.java

```diff
@@ -0,0 +1,73 @@
+package org.odk.collect.android.tasks.sms.models;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+/**
+ * Encapsulates all the data that's received when the SmsSentBroadcastReceiver
+ * is triggered. This is then passed to the SmsService and Notification Receiver for processing.
+ */
+public class SentMessageResult implements Parcelable {
+    private MessageResultStatus messageResultStatus;
+    private int messageId;
+    private String instanceId;
+
+    public MessageResultStatus getMessageResultStatus() {
+        return messageResultStatus;
+    }
+
+    public void setMessageResultStatus(MessageResultStatus messageResultStatus) {
+        this.messageResultStatus = messageResultStatus;
+    }
+
+    public int getMessageId() {
+        return messageId;
+    }
+
+    public void setMessageId(int messageId) {
+        this.messageId = messageId;
+    }
+
+    public String getInstanceId() {
+        return instanceId;
+    }
+
+    public void setInstanceId(String instanceId) {
+        this.instanceId = instanceId;
+    }
+
+
+    @Override
+    public int describeContents() {
+        return 0;
+    }
+
+    @Override
+    public void writeToParcel(Parcel dest, int flags) {
+        dest.writeInt(this.messageResultStatus == null ? -1 : this.messageResultStatus.ordinal());
+        dest.writeInt(this.messageId);
+        dest.writeString(this.instanceId);
+    }
+
+    public SentMessageResult() {
+    }
+
+    protected SentMessageResult(Parcel in) {
```

Comment:
Can be private.

Reply:
Note to @jd-alexander : 
It seems that many of these things @grzesiek2010 is pointing out you could see yourself with a good IDE.

Reply:
Yes, so true! It's after working on the Javarosa PR and you pointed it out I realized all the highlights by the IDE I wasn't paying attention to :see_no_evil: